src/Menu/SideDrawer.css

```diff
@@ -0,0 +1,105 @@
+.drawer {
+    margin-top: 10vh;
+    width: 250px ;
+    height: 90vh;
+    background: black;
+    color: white;
+ }
+
+.menu {
+    padding-left: 1rem;
+    line-height: 400%;
+    text-decoration: none;
+    text-align: left;
+ }
+
+.mainmenu{
+   font-size: 17px;
+   line-height: 400%;
+   padding-right: 80px;
+}
+
+.Collapsive_menu{
+   padding-left: 1rem;
+   line-height: 400%;
+   text-decoration: none;
+   text-align: center;
+   font-size: 20px;
+    display: block;  
+}
+
+.database_icon{
+   width: 30px;
+   height: 30px;
+   float: left;
+   margin-top: 20px;
+}
+
+.menu:hover {
+   background-color: gray;
+}
+
+.Collapsive_menu:hover {
+    background-color: gray;
+  }
+
+.menu_icon{
+   color:white;
```

Comment:
Make sure to maintain standards across files:
leave a space after colon in declarations

Reply:
ok

Reply:
Why there is no space after colon?

Reply:
Resolved